aisera/manifest.json

```
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ed21d816-fc99-4454-99d9-da6404d448a7",
+  "app_id": "aisera_aiops",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Aisera Aiops.",
+    "title": "Aisera",
+     "media": [
+      {
+        "media_type": "image",
+        "caption": "Aisera AiOps",
+        "image_url": "logos/aisera_logo.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Alerting",
+      "Category::Automation",
+      "Category::Issue Tracking",
+      "Category::Notification",
+      "Queried Data Type::Metrics",
+      "Queried Data Type::Logs",
+      "Queried Data Type::Traces",
```

```suggestion
      "Queried Data Type::Metrics",
      "Queried Data Type::Logs",
      "Queried Data Type::Traces",
```

To confirm, are you querying data or submitting data? Based on our previous conversations, this should be updated to "Submitted Data Type::Metrics".

---

This is right, We are querying the data from Datadog.